cuqi/model/_model.py

```diff
@@ -463,14 +473,67 @@ def get_matrix(self):
     def __matmul__(self, x):
         return self.forward(x)
 
+    def add_shift(self, shift):
+        """ Creates a ShiftedLinearModel with a fixed shift, i.e. model_shifted(x) = model(x) + shift. """
```

Comment:
In choosing our name, we may want to consider that a shift could also be referring to shifting the input before applying the model, i.e.  model_shifted(x) = model(x-shift), which is something we may need in the future.  In the context of a function of one variable, model(x-shift) would be a horizontal shift/translation, where model(x) + shift would be a vertical shift.

Reply:
I changed it to the operator "+"